src/Compilers/Core/Portable/CommandLine/AnalyzerConfig.SectionNameMatching.cs

```diff
@@ -3,18 +3,22 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Collections.Concurrent;
 using System.Collections.Immutable;
 using System.Diagnostics;
 using System.IO;
 using System.Text;
 using System.Text.RegularExpressions;
+using Microsoft.CodeAnalysis.Collections;
 using Microsoft.CodeAnalysis.PooledObjects;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis
 {
     public sealed partial class AnalyzerConfig
     {
+        internal static ConcurrentDictionary<string, Regex> RegexMap = [];
```

Comment:
Could be readonly if it doesn't get reassigned.
```suggestion
        internal readonly static ConcurrentDictionary<string, Regex> RegexMap = [];
```